### PR TITLE
add-scroll-to-top-feature(#54)

### LINKED ISSUE #54: [Feature] Add a scroll-to-top element to UX for longer page contexts
## Description

Many pages within Creative Commons websites have much longer pages, in these cases a `scroll-to-top` element would serve a valid purpose in certain contexts to aid in improved UX.

The element would likely not be present on smaller screens which already have some measure of functionality related to this baked into their operating system level UX; but for larger viewport sizes this could be much more helpful to include.

## Additional context
* https://github.com/creativecommons/cc-legal-tools-app/issues/474
* https://github.com/creativecommons/cc-resource-archive/pull/216

Solutions here should explore non-js interventions. CSS pseudo elements using z-index stack positions and variable percentage based heights would offer a means to "reveal" the button after a degree of scrolling, and to visually hide the element on pages which do not reach a particular length. Special care for keyboard accessibility should be taken under consideration to ensure that it can receive appropriate focus and order handling.

## Implementation

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by Asheklm6**
## Fixes

- Fixes #54 by @possumbilities

## Description

This PR implements a scroll-to-top button that becomes visible after the user scrolls down a certain distance on the page, improving navigation for users.

## Technical details

Added a scroll-to-top button in index.html.
Styled the button using CSS in style.css to ensure it appears fixed at the bottom right of the screen.
The button is initially hidden and designed to appear after scrolling down.

## Tests

Open index.html in a browser.
Scroll down the page to verify that the button appears after scrolling down

## Screenshots

Scroll before.
Screenshot (111)

Scroll After

Screenshot (112)

## Checklist

- [x ] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x ] My pull request targets the *default* branch of the repository (`main` or `master`).
- [ x] My commit messages follow [best practices][best_practices].
- [x ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ x] I added or updated documentation (if applicable).
- [ x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by SisiVero** [comment]
hi @Asheklm6 , well done. 
where you have "Fixes #[54] by @[asheklm6]", remove the "[" and "]" . just write #54 so as to link your PR to the issue #54 that you're about to fix.
Also instead of your username "@[asheklm6]" write @possumbilities as she is the one that opened the issue.
You're doing great. Well done.

**Comment by Asheklm6** [comment]
@SisiVero I've made the changes as you suggested. I've updated the PR description to remove the brackets around the issue number and replaced my username with @possumbilities. Thank you for your guidance . Please let me know if there's anything else I should adjust.

**Comment by SisiVero** [comment]
@Asheklm6 , well done.
Your PR details are okay now.
@possumbilities is in charge of reviewing PRs and will have a look at it by Monday.

File: _headers [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  X-Content-Type-Options: nosniff
+  Content-Security-Policy: default-src 'self'; img-src 'self' data:; script-src 'self'
+
+/index.html         # Applies specifically to index.html
+  Cache-Control: max-age=3600
+  Content-Security-Policy: default-src 'self'
+
+/images/*           # Applies to all files in the /images/ folder
+  Cache-Control: max-age=31536000
+  Access-Control-Allow-Origin: *
```
Comment (possumbilities) [inline _headers:12]: Changes here are unrelated to this PR

File: docs/index.html [hunk 2]
```diff
@@ -1025,5 +1025,13 @@ <h1>Name</h1>
 document.querySelectorAll("code.lang-html").forEach(el => el.innerText = el.innerHTML);
 </script>
 
+
```
Comment (possumbilities) [inline docs/index.html:1028]: extra space added.

File: docs/style.css [hunk 3]
```diff
@@ -26,4 +30,58 @@ span.distinct {
...
+    }
+  }      
+
+
+
+
+
+
+
+
```
Comment (possumbilities) [inline docs/style.css:87]: missing EOF character

**Comment by possumbilities** [comment]
Hi @Asheklm6 thank you for working on this! 

The feature in question is for the Vocabulary UX, not the docs. You've added this element to the docs instead. 

Your solution here would need to be added to the Vocabulary contexts themselves, not the documentation.
